### PR TITLE
Make hack warning to info and suppress InferenceValidator error

### PR DESCRIPTION
**Pull request by topnessman**

### CODE REVIEW

File: src/checkers/inference/VariableAnnotator.java [hunk 1]
```diff
@@ -825,7 +825,7 @@ private void handleClassDeclaration(AnnotatedDeclaredType classType, ClassTree c
      */
     private AnnotationLocation createImpliedExtendsLocation(ClassTree classTree) {
         // TODO: THIS CAN BE CREATED ONCE THIS IS FIXED: https://github.com/typetools/annotation-tools/issues/100
```
Comment (wmdietl) [inline src/checkers/inference/VariableAnnotator.java]: The references issue has been fixed.
I assume we currently don't need annotations on extends clauses.

Comment (topnessman) [reply src/checkers/inference/VariableAnnotator.java]: Should we simply remove the TODO and the logger warning OR fix the caller:
https://github.com/opprop/checker-framework-inference/blob/4101d926b6f59bf945a3ee07574ce10770fb9bd8/src/checkers/inference/VariableAnnotator.java#L772
Specifically method `VariableAnnotator#handleClassDeclaration`?

Comment (topnessman) [reply src/checkers/inference/VariableAnnotator.java]: Keep as warning.

File: src/checkers/inference/InferenceValidator.java [hunk 2]
```diff
@@ -202,8 +202,10 @@ public Void visitDeclared(AnnotatedDeclaredType type, Tree tree) {
             // ParameterizedTypeTree));
             break;
         default:
-            System.err.printf("TypeValidator.visitDeclared unhandled tree: %s of kind %s\n",
-                            tree, tree.getKind());
+            // This caused too many error outputs. To be consistent with BaseTypeValidator, below
```
Comment (topnessman) [inline src/checkers/inference/InferenceValidator.java]: UNdo this.

**Comment by topnessman** [comment]
Ready to review.

**Pull request merged**